src/main/java/frc/robot/subsystems/ClawSubsystem.java

```diff
@@ -43,10 +62,18 @@ public void clamp() {
   }
 
   public void release() {
-
     motleft.setInverted(true);
     motright.setInverted(false);
-    clawGroup.set(-0.4);
+    if (getrightenc() >= 0) {
```

Comment:
TODO: We'll need to make sure that encoder values increase as the claws move inward.  (may need to invert the values)